spec/template.dd

```diff
@@ -966,6 +966,18 @@ $(H2 $(LNAME2 function-templates, Function Templates))
         }
         ------
 
+    $(P It is lowered to:)
+
+        ------
+        template $(CODE_HIGHLIGHT Square)(T)
+        {
+            T $(CODE_HIGHLIGHT Square)(T t)
```

Comment:
Reminder @ self: check the PDF output of this. The same pattern is used in the code block above, though.